root/utility/voting.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+export function isInvolved(election: AutoEditorElectionT, user: ?EditorT) {
+  return !!user && (
+    election.proposer.id === user.id ||
+    election.candidate.id === user.id ||
+    (election.seconder_1 && election.seconder_1.id === user.id) ||
+    (election.seconder_2 && election.seconder_2.id === user.id)
+  );
+}
+
+export function votesVisible(election: AutoEditorElectionT, user: ?EditorT) {
```

Comment:
This isn't equivalent to the old `can_see_vote_count`, because it returns `true` if the election is pending (still awaiting seconders). So the templates now show a vote tally even if the vote didn't start yet.